engine/src/test/java/net/jqwik/api/EMailsTests.java

```diff
@@ -0,0 +1,204 @@
+package net.jqwik.api;
+
+import com.tngtech.archunit.thirdparty.com.google.common.net.*;
+
+import net.jqwik.api.statistics.*;
+
+import org.assertj.core.api.*;
+
+import java.util.stream.*;
+
+class EMailsTests {
+
+	@Property(tries = 10, edgeCases = EdgeCasesMode.NONE)
```

Comment:
Since you run all properties with changed edgeCasesMode you can use `@ProperyDefaults(edgeCases=NONE)` on the class level.